x-pack/legacy/plugins/siem/public/components/url_state/index.test.tsx

```diff
@@ -104,59 +110,26 @@ describe('UrlStateContainer', () => {
         );
       });
 
-      describe('kqlQuery action is called with correct data on component mount', () => {
-        const serializedFilterQuery = {
-          kuery: {
-            expression: 'host.name:"siem-es"',
-            kind: 'kuery',
-          },
-          serializedQuery:
-            '{"bool":{"should":[{"match_phrase":{"host.name":"siem-es"}}],"minimum_should_match":1}}',
-        };
+      describe('appQuery action is called with correct data on component mount', () => {
         test.each(testCases.slice(0, 4))(
           ' %o',
           (page, namespaceLower, namespaceUpper, examplePath, type, pageName, detailName) => {
             mockProps = getMockPropsObj({ page, examplePath, namespaceLower, pageName, detailName })
               .relativeTimeSearch.undefinedQuery;
             mount(<HookWrapper hookProps={mockProps} hook={args => useUrlStateHooks(args)} />);
-            const functionName =
-              namespaceUpper === 'Network'
-                ? mockApplyNetworkFilterQuery
-                : mockApplyHostsFilterQuery;
             // @ts-ignore property mock does not exists
```

Comment:
Doesn't look like any of these `@ts-ignore`'s are needed in this file anymore. I was able to remove them without error.